### ISSUE TITLE
[SPIKE] A more generic approach for exposing Accessibility Notifications

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3339]
Scope
---
There are several components like `Button`, `Toast`, `Textfield`, `Banner`, `Inline Notification`, etc. which expose more or less relevant notifications to the user (status and alerts). Regarding of the context and the severity of the notification, they should/must also be announced by screen reader software.

The support of those components regarding accessibility compliance is slightly different. Some components have good support (e.g. error handling of form input components) and some lack support (e.g. `Toast`). This is mainly because of not rendering an HTML element container initially with `aria-live` attribute, but rendering it on-the-fly which is not supported by assistive technologies.

The goal of this SPIKE is to research and evaluate possibilities of generalizing notifications for assistive technologies.

Out of Scope
---

Notes  
---
- The support of ARIA live-regions is handled differently in various screen reader software
- Think about creating a dedicated "accessibility notification" component which handles all notifications in one place.
- Article: https://www.sarasoueidan.com/blog/accessible-notifications-with-aria-live-regions-part-1/ 
- Article: https://www.sarasoueidan.com/blog/accessible-notifications-with-aria-live-regions-part-2/

Acceptance criteria
---
- [ ] Solution to expose notifications also for assistive technologies equal to the visual counter-part

Subtasks
---
- [ ] Task
